sirbot/slack/store/user.py

```diff
@@ -56,16 +56,23 @@ class UserStore(SlackStore):
     def __init__(self, client, facades, refresh=3600):
         super().__init__(client, facades, refresh)
 
-    async def all(self):
+    async def all(self, fetch=False, deleted=False):
         db = self._facades.get('database')
-        data = await database.__dict__[db.type].user.get_all(db)
-
+        db_data = await database.__dict__[db.type].user.get_all(db)
```

Comment:
you should query the db only if `fetch` is `False` there is no need to do it when it's `True`